doc/manual/configuration.rst

```diff
@@ -852,7 +852,7 @@ If the package is built from multiple modules you can give a list of SCMs::
             dir: src/bar
 
 There are three common (string) attributes in all SCM specifications: ``scm``,
-``dir`` and ``if``. Additionally to the string type, the ``if`` property may
+``dir`` (\*) and ``if``. Additionally to the string type, the ``if`` property may
```

Comment:
Looks like the `cvs` SCM still missing this change...

Reply:
you are right.. The unit test will also follow..